modules/adnuntiusRtdProvider.js

```diff
@@ -0,0 +1,97 @@
+import { getGlobal } from '../src/prebidGlobal.js';
+import { submodule } from '../src/hook.js'
+import { logError, logInfo } from '../src/utils.js'
+import { ajax } from '../src/ajax.js';
+
+const GVLID = 855;
+
+function init(config, userConsent) {
+  if (!config.params) return false
+  if (!config.params.providers) return false
```

Comment:
Can we consolidate this into one conditional i.e. if (!config.params || !config.params.providers) return false;

Reply:
Fixed in new pull request. :)